staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -363,6 +363,14 @@ type PersistentVolumeSpec struct {
 	// This field influences the scheduling of pods that use this volume.
 	// +optional
 	NodeAffinity *VolumeNodeAffinity `json:"nodeAffinity,omitempty" protobuf:"bytes,9,opt,name=nodeAffinity"`
+	// Name of VolumeAttributesClass to which this persistent volume belongs. Empty value
+	// is not allowed. Use nil to indicate that this volume does not belong to any
+	// VolumeAttributesClass. This field is mutable and can be changed by the CSI driver
```

Comment:
```suggestion
	// is not allowed. When this field is not set, it indicates that this volume does not belong to any
	// VolumeAttributesClass. This field is mutable and can be changed by the CSI driver
```

(Since nil a go specific concept and this comment is exposed as API documentation, which not language specific)